src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -172,20 +245,15 @@ export default {
 			}
 			this.item.edit.reset(e)
 		},
-		// This is used to decide which outter element type to use
+		// This is used to decide which outer element type to use
 		// li or router-link
-		navElement(item) {
-			if (item.router) {
-				let exact = item.router.exact
-				if (typeof item.router.exact === 'undefined') {
-					exact = true
-				}
-
+		navElement() {
+			if (this.route) {
 				return {
 					is: 'router-link',
 					tag: 'li',
-					to: item.router,
-					exact
+					to: this.route,
```

Comment:
I would rather have two AppNavigationItem with one mixin for things that are common to both.
AppNavigationItem and AppNavigationItemRouter ? What do you think?
That way this is cleaner?

Reply:
Or I actually am wondering if using route or exact on the `AppNavigationItem` works?
If so it's easier to have directly a check for the route or exact prop and apply it there?

That way we could directly do `<AppNavigationItem route="contacts" :exact="true" />`

Reply:
Yes, we can do that.
have a to props and use it there, it will be better.
```suggestion
					to: this.to,
```